ipatests/test_integration/test_automember.py

```diff
@@ -211,6 +211,11 @@ def test_modify_user_entry_unmembership_disabled(self):
             assert self.is_user_member_of_group(user2, group2)
             assert not self.is_user_member_of_group(user2, group1)
 
+            msg = ('IMPORTANT: In case of a high number of users, hosts or '
+                   'groups, the operation may require high CPU usage.')
+            result = self.master.run_command(['ipa', 'automember-rebuild'])
```

Comment:
You don't need to call the command a second time, it was already executed in line 209 (and with the expected arguments, which are missing in your line 216). Just store the result of the command in line 209 (replace self.master.run_command with result = self.master.run_command)